recipes/nas/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+
+class NasRecipe(ConanFile):
+    name = "nas"
+    description = "The Network Audio System is a network transparent, client/server audio transport system."
+    topics = ("audio", "sound")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.radscan.com/nas.html"
+    license = "Unlicense"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Recipe supports Linux only")
+
+    def requirements(self):
+        self.requires("xorg/system")
+
+    def build_requirements(self):
+        self.build_requires("bison/3.7.1")
+        self.build_requires("flex/2.6.4")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version][0], strip_root=True, destination=self._source_subfolder)
+        tools.download(filename="LICENSE", **self.conan_data["sources"][self.version][1])
+
+    def build(self):
+        with tools.chdir(self._source_subfolder):
+            env_build = AutoToolsBuildEnvironment(self)
+            self.run("xmkmf")
```

Comment:
We need to go deeper:
https://gitlab.freedesktop.org/xorg/util/imake

Reply:
I'll work on a recipe for this.

Reply:
Down the rabbit hole 😄

Reply:
First step: #6390